### DISCUSSION TITLE
Pikka now Crashes on Big Sur when open the color picker, and then click on the Preview app to choose a color.

### DISCUSSION THREAD

**Discussion by paulbradyhanleywood** [rsayfutdinov/Pikka#46] [General]
I suspect this is a permissions / OS security issue?  
(Pikka runs fine on my M1 on the same iCloud account.)

I've had Pikka running for about a year.  This is a fairly recent problem.  Here are my machine's stats...

MacBook Pro (Retina, 15-inch, Mid 2014)
Big Sur v11.7.6
16GB
Intel Iris Pro 1536MB

Any ideas?  Thanks!

**Comment by hirofyam** [comment]
I am in trouble with the same disability.
Big Sur 11.7.7
MacBook Pro (2020,13-inch)
Intel Core7 16G

**Comment by rsayfutdinov** [comment]
I'm looking into it. Will try to fix this asap.
